### PR TITLE
Add arm64 centos7

### PR DESCRIPTION
**Pull request by DrewScoggins**
This adds a Arm64 Centos 7 image that matches the one that we use for Linux Amd64 optimization data gathering.

### CODE REVIEW

File: src/centos/manifest.json
Comment (mthalman) [inline src/centos/manifest.json:32]: Not needed since there is no dependent Dockerfile that needs to reference this tag.

**Review by mthalman — COMMENTED** [review]
This will also require updating the build timeouts to account for arm. Please add 
 the following parameter, `linuxArmBuildJobTimeout: 210`, to https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/31c08723eca277df5b0e044b261d438a7578f5c7/eng/pipelines/dotnet-buildtools-prereqs-centos-pr.yml#L24-L27 and https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/31c08723eca277df5b0e044b261d438a7578f5c7/eng/pipelines/dotnet-buildtools-prereqs-centos.yml#L24-L27

File: eng/common/templates/stages/dotnet/build-test-publish-repo.yml
Comment (mthalman) [inline eng/common/templates/stages/dotnet/build-test-publish-repo.yml:16]: The changes in this file are not needed.

File: eng/pipelines/dotnet-buildtools-prereqs-centos.yml
Comment (mthalman) [inline eng/pipelines/dotnet-buildtools-prereqs-centos.yml:28]:
```suggestion
    linuxArmBuildJobTimeout: 210
```

The same parameter needs to be added to: https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/31c08723eca277df5b0e044b261d438a7578f5c7/eng/pipelines/dotnet-buildtools-prereqs-centos-pr.yml#L24-L27

Comment (DrewScoggins) [reply eng/pipelines/dotnet-buildtools-prereqs-centos.yml:28]: OK, I was wondering where I needed to add that. I will make these changes.

**Comment by DrewScoggins** [comment]
I believe I have now addressed all of the feedback. I am going to make sure that the build of the new image completes, and then hopefully we should be good to go.

**Review by mthalman — APPROVED** [review]

**Comment by DrewScoggins** [comment]
I don't have write access in the this repo. So when it finishes, I will likely ping you to hit the merge button :)

**Comment by DrewScoggins** [comment]
These changes also timeout, I am going to go back to the previous changes, as they at least allowed the build to complete.

File: eng/pipelines/dotnet-buildtools-prereqs-centos.yml
Comment (mthalman) [inline eng/pipelines/dotnet-buildtools-prereqs-centos.yml:28]: Looks like the timeout needs to be increased.
```suggestion
    linuxArmBuildJobTimeout: 270
```

Comment (mthalman) [reply eng/pipelines/dotnet-buildtools-prereqs-centos.yml:28]: This same timeout parameter needs to be added here:

https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/26b5ef43b338cd9f130ec4ee03994886947f8896/eng/pipelines/dotnet-buildtools-prereqs-eng.yml#L25-L30

**Comment by mthalman** [comment]
@DrewScoggins - This change still needs to be made: https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/653#discussion_r940202043

**Comment by DrewScoggins** [comment]
Yeah, I just made the change.

**Comment by DrewScoggins** [comment]
@mthalman Looks like this has passed. I am going to up the timeout to 300 minutes however, as the current runs take about 276 of the 280 allotted minutes. But once I make the change, could you click the merge button?

**Pull request merged**
